elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/resolvers/DailyAverageScorePerPeriodResolver.java

```diff
@@ -52,6 +52,7 @@ public QueryPlan resolve(MetricProjection projection) {
                         .expression(projection.getExpression())
                         .columnType(projection.getColumnType())
                         .valueType(projection.getValueType())
+                        .arguments(projection.getArguments())
```

Comment:
Can we try a filter (with arguments) on a query with a two pass aggregation like daily average score and make sure it works?